plugin-templates/blocks/theme-navigation/edit.tsx

```diff
@@ -64,14 +65,22 @@ export default function Edit({
 
   // Loading menu locations.
   if (!Array.isArray(menuLocations)) {
-    return <Spinner />;
+    return (
+      <div {...blockProps}>
+        <Spinner />
+      </div>
+    );
   }
 
   // No menu locations have been registered.
   if (menuLocations.length === 0) {
     return (
       <div {...blockProps}>
-        <h3>{ __('Please register menu locations.', 'create-wordpress-plugin') }</h3>
+        <Placeholder
+          icon={ menu }
+          label={ __('Theme Navigation', 'create-wordpress-project') }
+          instructions={ __('Please register menu locations.', 'create-wordpress-project') }
```

Comment:
The textdomain in the `block.json` is `create-wordpress-plugin` but here it is `create-wordpress-project`.